config/routes.rb

```diff
@@ -13,6 +13,10 @@ def set_up_flipper
     omniauth_callbacks: 'users/omniauth_callbacks'
   }
 
+  authenticate :user, lambda { |u| u.super_admin? } do
```

Comment:
This is going to conflict with the roles PR, FYI. It won't automatically fix itself or have a merge conflict, so maybe rebase off the roles PR to do this?

Reply:
I guess whichever one goes out first the other one should immediately rebase off it.